onyxia-api/src/main/java/fr/insee/onyxia/api/controller/api/mylab/MyLabController.java

```diff
@@ -190,11 +190,11 @@ public UninstallService destroyApp(
             @Parameter(hidden = true) Region region,
             @Parameter(hidden = true) Project project,
             @RequestParam(value = "path", required = false) String path,
-            @RequestParam(value = "bulk", required = false) boolean bulk)
```

Comment:
Should we just remove this parameter, @olevitt ? Onyxia-web doesn't use it, e.g. for the 'delete all' onyxia-web send multiple delete calls (one for each service)

Reply:
I think it's ok to keep it as it may be useful for other clients (such as an admin tool that would cleanup)